### ISSUE TITLE
Blob can stack multiple strong blob walls on one tile

### ISSUE THREAD

**Issue by ghost** [d3athrow/vgstation13#21930]
Blob can stack multiple strong blob walls on one tile
[exploitable]

**Comment by Killette2** [comment]
it's uh feature

**Comment by DeityLink** [comment]
Yeah nah, that needs to go. Wish we had some Steps to Reproduce though. I've seen it happen twice yesterday but I don't know how what caused it exactly. One theory is that it can happen when multiple overminds are present.

**Comment by MadmanMartian** [comment]
In my efforts to reproduce this, I have managed to make a null blob through an explosion

![image](https://user-images.githubusercontent.com/30557196/53752128-b6dd6780-3ea5-11e9-8634-2ca7ac26d6b6.png)

**Comment by MadmanMartian** [comment]
Update: Oh lawd turns out these null blobs can be turned into unlimited strong walls

![image](https://user-images.githubusercontent.com/30557196/53752176-db394400-3ea5-11e9-8c01-41362bfd6de7.png)

**Comment by MadmanMartian** [comment]
Jumping to the 'null blobs' leads me to the 3 blobs that were surrounding it, so maybe the veins aren't being deleted?

**Comment by Lunstra** [comment]
Maybe the null blobs might also be the cause of the phantom impassable invincible blob thing that's happened before too?

**Comment by DeityLink** [comment]
Probably yeah. That wall of blobs on the shuttle was exclusively made of strong blob.
